lib/hanami/assets.rb

```diff
@@ -32,15 +32,11 @@ def initialize(config:)
     # @api public
     def [](path)
       asset_attrs =
-        if manifest?
-          manifest.fetch(path).transform_keys(&:to_sym).tap { |attrs|
-            # The `url` attribute we receive from the manifest is actually a path; rename it as
-            # such so our `Asset` attributes make more sense on their own.
-            attrs[:path] = attrs.delete(:url)
-          }
-        else
-          {path: config.path_prefix + SEPARATOR + path}
-        end
+        manifest.fetch(path).transform_keys(&:to_sym).tap { |attrs|
```

Comment:
@timriley Can we save time by not doing `to_sym`?

Reply:
@jodosha unfortunately we need these for the keyword args that we use for `Asset#initialize` — splatting a hash with string keys won't match up to those keyword args.

The alternative would be to let `Asset#initialize` receive an opaque hash and expect string keys on it, but I think that approach would be one that more likely leads to bugs if we're not careful.